pkg/config/remote/service/service.go

```diff
@@ -292,6 +307,38 @@ func (s *Service) Stop() error {
 	return s.db.Close()
 }
 
+func (s *Service) pollOrgStatus() {
+	response, err := s.api.FetchOrgStatus(context.Background())
+	if err != nil {
+		// Unauthorized and proxy error are caught by the main loop requesting the latest config,
+		// and it limits the error log.
+		if !errors.Is(err, api.ErrUnauthorized) && !errors.Is(err, api.ErrProxy) {
+			log.Errorf("Could not refresh Remote Config: %v", err)
+		}
+		return
+	}
+
+	// Print info log when the new status is different from the previous one, or if it's the first run
+	if s.previousOrgStatus == nil || s.previousOrgStatus.Enabled != response.Enabled {
+		if !response.Enabled {
+			log.Infof("This org does not have Remote Configuration enabled, please follow the documentation to enable it.")
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 Nitpick:** ‏Not sure we want to use the term "org" here is the log is customer-facing. What about something in the likes of "Remote Configuration isn't enabled, please follow the documentation to enable it"? I don't really have a strong opinion here though, and we may be using "org" elsewhere in the logs.